Dockerfile.integration

```
@@ -2,13 +2,12 @@
 # and are in no way endorsed by Headscale's maintainers as an
 # official nor supported release or distribution.
 
-FROM docker.io/golang:1.25-bookworm
+FROM docker.io/golang:1.25-trixie
 ARG VERSION=dev
 ENV GOPATH /go
 WORKDIR /go/src/headscale
 
-RUN apt-get update \
-  && apt-get install --no-install-recommends --yes less jq sqlite3 dnsutils \
+RUN apt-get --update install --no-install-recommends --yes less jq sqlite3 dnsutils \
   && rm -rf /var/lib/apt/lists/* \
   && apt-get clean
```

FWIW, `apt-get clean` is a no-op in the Debian (and Ubuntu) base images

Perhaps confusingly in relation to that, APT in Trixie now supports a separate `apt-get dist-clean` command that's similar to this `rm -rf`, but better (because it keeps the cryptographic downgrade-attack-preventing verification bits).

```suggestion
  && apt-get dist-clean
```